awx/ui_next/src/screens/Organization/shared/OrganizationForm.jsx

```diff
@@ -93,7 +93,7 @@ function OrganizationFormFields({
         onBlur={() => executionEnvironmentHelpers.setTouched()}
         value={executionEnvironmentField.value}
         onChange={value => executionEnvironmentHelpers.setValue(value)}
-        popoverContent={t`Select the default execution environment for this organization.`}
+        popoverContent={t`The Execution Environment that will be used for jobs inside of this organization. This will be used a fallback when an EE has not been explicitly assigned at the Project, Job Template or Workflow level.`}
```

Comment:
In general, Execution Environment is not capitalized... but neither would be Project, Job Template, or Workflow. So... meh.

I would not use 'EE' - would spell it out.

Reply:
@wenottingham updated.